### PR TITLE
Add option that allows to specify what IP to listen on

### PR DESCRIPTION
**Pull request by Martin8412**
This PR adds an option to specify which IP the server should listen on.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -84,6 +89,7 @@ var (
 
 func init() {
 	flags := rootCmd.Flags()
+	flags.StringVarP(&host, "host", "", "0.0.0.0", "IP address to listen on. Default is all interfaces.")
```
Comment (brendan-ward) [inline main.go:92]:
```suggestion
	flags.StringVar(&host, "host", "0.0.0.0", "IP address to listen on. Default is all interfaces.")
```

File: main.go [hunk 2]
```diff
@@ -107,6 +113,10 @@ func init() {
 	flags.StringVar(&sentryDSN, "dsn", "", "Sentry DSN")
 	flags.BoolVarP(&verbose, "verbose", "v", false, "Verbose logging")
 
+	if env := os.Getenv("INTERFACE"); env != "" {
```
Comment (brendan-ward) [inline main.go:116]:
```suggestion
	if env := os.Getenv("HOST"); env != "" {
```
(for consistency with flag)

File: main.go [hunk 3]
```diff
@@ -435,12 +445,12 @@ func serve() {
...
 			tlsListener := tls.NewListener(listener, server.TLSConfig)
 
-			fmt.Printf("HTTPS server started on port %v\n", port)
+			fmt.Printf("HTTPS server started on %v:%v\n", host, port)
 			log.Fatal(server.Serve(tlsListener))
 		}
 	default:
 		{
-			fmt.Printf("HTTP server started on port %v\n", port)
+			fmt.Printf("HTTPS server started on %v:%v\n", host, port)
```
Comment (brendan-ward) [inline main.go:453]:
```suggestion
			fmt.Printf("HTTP server started on %v:%v\n", host, port)
```

**Review by brendan-ward — CHANGES_REQUESTED** [review]
Thanks for submitting this @Martin8412 !

A few minor changes, then this looks good to go.

**Comment by Martin8412** [comment]
I've updated according to your review

**Comment by brendan-ward** [comment]
@Martin8412 looks like maybe your force-push reverted an earlier commit?  The PR is only showing your original changes, not your latest...

**Comment by brendan-ward** [comment]
thanks @Martin8412 🎉

**Pull request merged**